packages/ketcher-core/src/domain/entities/sgroup.ts

```diff
@@ -348,12 +351,24 @@ export class SGroup {
 
     let braketBox: Box2Abs | null = null
     const contentBoxes: Array<any> = []
+    const getAtom = (aid) => {
+      if (remol && render) {
+        return remol.atoms.get(aid)
+      }
+      return mol.atoms.get(aid)
+    }
     atoms.forEach((aid) => {
-      const atom = mol.atoms.get(aid)
-      const pos = new Vec2(atom.pp)
+      const atom = getAtom(aid)
       const ext = new Vec2(0.05 * 3, 0.05 * 3)
-      const bba = new Box2Abs(pos, pos).extend(ext, ext)
-      contentBoxes.push(bba)
+      let position
+      let structBoundingBox
+      if ('getVBoxObj' in atom && render) {
```

Comment:
Hmm, why is this check for 'getVBoxObj' needed?

Reply:
replaced with `instanceof` check